tests/test_apt-cacher-ng-formula.py

```diff
@@ -2,27 +2,90 @@
 # -*- coding: utf-8 -*-
 
 """
-test_apt-cacher-ng-formula
+test_apt-cacher-ng_formula
 ----------------------------------
 
-Tests for `apt-cacher-ng-formula` module.
+Tests for `apt_cacher_ng_formula` module.
 """
 
 import unittest
+import salt.utils.templates
+import salt.template
+import salt.loader
+import salt.renderers.jinja
+import pprint
+import logging
+import sys
+logging.basicConfig(
+    stream=sys.stderr,
+    level=logging.DEBUG
+)
 
-from apt-cacher-ng-formula import apt-cacher-ng-formula
+import salt.renderers.jinja
+import salt.transport
+import pprint
+from mock import MagicMock
+import mock
 
 
-class TestApt-cacher-ng-formula(unittest.TestCase):
+
+
+class TestAptCacherNgFormula(unittest.TestCase):
 
     def setUp(self):
-        pass
 
-    def test_something(self):
-        pass
+        salt.renderers.jinja.__salt__= MagicMock()
+        salt.renderers.jinja.__grains__= MagicMock()
+        salt.renderers.jinja.__opts__= {
+            'cachedir' : "/tmp/cachedir",
```

Comment:
As far as doing this right the first time, I think it would be good to utilize `mkdtemp` whether or not the temp directory is used.